pbf/views.py

```diff
@@ -345,42 +345,92 @@ def view_game(request, game_id):
                 add_log_msg(game, text=f'New screenshot uploaded.', images='.' + game.screenshot2.url)
                 return redirect(reverse('view_game', args=[game.id]))
 
+    spirits_by_expansion = {
+        # Short name, full name, aspects (if any)
+        # Within an expansion, sorted alphabetically
+        #
+        # Should we sort by complexity?
+        # Don't think so; not everyone knows the complexities by heart,
+        # and so the ordering would look haphazard to those who don't know them.
+        #
+        # Similarly, aspects are going to be grouped with their spirit
+        # regardless of which expansion the aspect was released in,
+        # because not everyone is going to know which expansion has which aspects.
+        'Spirit Island (base)': [
+            ('Bringer', 'Bringer of Dreams and Nightmares', ('Enticing', 'Violence')),
+            ('Lightning', "Lightning's Swift Strike", ('Immense', 'Pandemonium', 'Sparking', 'Wind')),
+            ('Ocean', "Ocean's Hungry Graps", ('Deeps',)),
```

Comment:
Graps? what is a Grap?

Reply:
There's always bound to be one when writing this much 😂